pulp_smash/tests/pulp3/pulpcore/api_v3/test_crud_repos.py

```diff
@@ -18,14 +18,13 @@ class CRUDRepoTestCase(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
         """Create class-wide variables."""
-        cls.auth = get_auth()
         cls.cfg = config.get_config()
         cls.repo = {}
 
     def setUp(self):
         """Create an API client."""
         self.client = api.Client(self.cfg, api.code_handler)
-        self.client.request_kwargs['auth'] = self.auth
+        self.client.request_kwargs['auth'] = get_auth()
```

Comment:
This is the kind of code that you can now write with a cache in hand.

Reply:
@Ichimonji10, pretty cool approach.

I verified the expiration time for the token in the documentation, and if does not specified will be set to **14** days.

[JWT Authentication](http://docs.pulpproject.org/en/3.0/nightly/integration_guide/rest_api/authentication.html#jwt-authentication)

Reply:
Cool. These tokens are stored in memory for the duration of a single test runner run, which is way less than 14 days, so this should be safe. Thanks for verifying.